packages/nc-gui/pages/signin.vue

```diff
@@ -114,6 +116,7 @@ function resetError() {
             <a-form-item :label="$t('labels.password')" name="password" :rules="formRules.password">
               <a-input-password
                 v-model:value="form.password"
+                autocomplete="password"
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete#values), `autocomplete="password"` is not valid. It can either be `"new-password"` or `"current-password"`.

In this case it should be **`autocomplete="current-password"`**.